lib/index.js

```diff
@@ -53,3 +66,21 @@ function calculateTotalRemaining(consumed, goal) {
   var caloriesRemainingCell = $('#totals-table tr:nth-child(3) td:nth-child(2)');
   caloriesRemainingCell.text(caloriesRemaining);
 }
+
+function formatTotalRemaining() {
+  var remainingCaloriesCell = $('#totals-table tbody tr:nth-child(3) td:nth-child(2)')
+  var remainingCalories = parseInt(remainingCaloriesCell.text());
+
+  formatRemainingCell(remainingCaloriesCell);
+}
+
+function formatRemainingCell(cell) {
+  var remainingCalories = parseInt(cell.text());
+
+  if(remainingCalories >= 0) {
+    cell.css("color", "green");
+  }
+  else {
+    cell.css("color", "red");
+  }
```

Comment:
This absolutely gets the job done, but I usually prefer to put my `color` definitions in the CSS file, and just change the class here. Like `cell.addClass('under-goal')`